lib/passenger.rb

```diff
@@ -3,21 +3,41 @@
 module RideShare
   class Passenger < CsvRecord
     attr_reader :name, :phone_number, :trips
-
+    
     def initialize(id:, name:, phone_number:, trips: nil)
       super(id)
-
+      
       @name = name
       @phone_number = phone_number
       @trips = trips || []
     end
-
+    
     def add_trip(trip)
       @trips << trip
     end
-
+    
+    def net_expenditures
+      total_expenditure = 0
+      trips.each do |trip|
+        if trip.cost.nil?
+          total_expenditure += 0
+        else
+          total_expenditure += trip.cost
+        end
+      end
+      return total_expenditure
+    end 
+    
+    def total_time_spent 
```

Comment:
I guess I'm confused about what we were supposed to do; we thought that by having the lines
```
if trip.cost.nil?
          total_expenditure += 0
```
that we were telling the program that if the trip cost was nil, which it would only be for an in-progress trip, then ignore it—add zero, thereby essentially doing nothing. That was our thought process anyway.